src/posix/platform/openthread-posix-config.h

```diff
@@ -355,7 +355,7 @@
  *
  */
 #ifndef OPENTHREAD_POSIX_CONFIG_PRODUCT_CONFIG_FILE
-#define OPENTHREAD_POSIX_CONFIG_PRODUCT_CONFIG_FILE "src/posix/platform/openthread.conf.example"
+#define OPENTHREAD_POSIX_CONFIG_PRODUCT_CONFIG_FILE "/tmp/openthread.conf.example"
```

Comment:
It seems the macro `OPENTHREAD_POSIX_CONFIG_PRODUCT_CONFIG_FILE` only contains power related config and it's only used in `tools/ot-fct/cli.hpp` and `src/posix/platform/power_updater.hpp`. Could the macro name be more specific?

Reply:
The config file only contains the power related configurations now. But it doesn't mean the config file can only contain power related configurations, we can add other configurations for products later, such as, EUI64, supported channel mask and preferred channel mask.